core/src/main/scala/cats/syntax/xort.scala

```diff
@@ -0,0 +1,39 @@
+package cats
+package syntax
+
+import cats.data.XorT
+
+trait XorTSyntax extends XorTSyntax1 {
+  implicit def catsSyntaxXorT[F[_]: Functor, A](fa: F[A]): XorTOps[F, A] = new XorTOps(fa)
+}
+
+trait XorTSyntax1 {
+  implicit def catsSyntaxUXorT[FA](fa: FA)(implicit U: Unapply[Functor, FA]): XorTOps[U.M, U.A] =
+    new XorTOps[U.M, U.A](U.subst(fa))(U.TC)
+}
+
+final class XorTOps[F[_]: Functor, A](val fa: F[A]) {
```

Comment:
Should we call this `XorTFunctorOps` or something? We could conceivably want to add `XorT`-related syntax onto types other than `F[A]`. I think it should still be mixed in via `XorTSyntax` though.